docs/layers.md

```diff
@@ -1093,10 +1109,11 @@ To resolve inconsistency in data tagging in OpenStreetMap we normalize several o
 * `water_point`
 * `water_slide`
 * `water_tower`
-* `water_well`
+* `water_well` - A location where water can be extracted from the ground. If known, `kind_detail` will be given as `drinking_water` or `not_drinking_water`. If information about pumping is known, the `pump` property will be one of `powered`, `manual` or `no`.
```

Comment:
Please consolidate the pump into the kind_detail so the icon lookup and be straight instead of need client-side logic.

Reply:
Ah, oops. I broke this out separately because I thought it looked awkward in a single tag. I've merged it all back in dcce2ee.